docs/DeveloperGuide.md

```diff
@@ -139,7 +139,7 @@ The `Model` component,
 
 ### Storage component
 
-**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storage/Storage.java)
+**API** : [`Storage.java`](https://github.com/AY2425S1-CS2103-F10-3/tp/blob/master/src/main/java/seedu/address/model/Model.java)
```

Comment:
Is this supposed to be storage instead of model?

Reply:
Yup, sorry. Just fixed it in new commit